gitautosync/cli.py

```diff
@@ -0,0 +1,15 @@
+import argparse
+from .pull_from_remote import pull_from_remote
+
+
+def main():
+    """
+    Synchronizes a github repository with a local repository.
+    Automatically deals with conflicts and produces useful output to stdout.
+    """
+    parser = argparse.ArgumentParser(description='Synchronizes a github repository with a local repository.')
+    parser.add_argument('--git-url', default='https://github.com/data-8/summer.git', help='Url of the repo to sync')
```

Comment:
Don't set this to be a default - make this a required param. So users should be able to do:

gitautosync <repo-url> <repo-dir>

Branch name can be an optional param. Should default to master tho, since that's what git defaults to.